arangod/RocksDBEngine/RocksDBTypes.h

```diff
@@ -79,9 +79,6 @@ enum class RocksDBLogType : char {
   SinglePut = '?',
   SingleRemove = '@',                  // <- deprecated
   DocumentRemoveAsPartOfUpdate = 'A',  // <- deprecated
-#ifdef USE_IRESEARCH
-  IResearchLinkDrop = 'C',
```

Comment:
Let's keep at least add a comment here that the marker character type 'C' was once used and should not be reused in the near future for anything else - otherwise we may have ambiguous data while upgrading databases or when someone is not following upgrade procedures etc.

Reply:
done